hack/verify-flags/exceptions.txt

```diff
@@ -5,11 +5,13 @@ cluster/aws/templates/configure-vm-aws.sh:  # We set the hostname_override to th
 cluster/aws/templates/configure-vm-aws.sh:  api_servers: '${API_SERVERS}'
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "hostname_override"
 cluster/aws/templates/configure-vm-aws.sh:  env-to-grains "runtime_config"
+cluster/aws/templates/configure-vm-aws.sh:  kubelet_api_servers: '${KUBELET_APISERVER}'
```

Comment:
It did fail PR validation because of this, which is the reason I made the change here (following the instructions of the PR validation result).

I want to clarify that the reason we need to make this change is because we are adding a flag "--kubelet-api-servers" in hacks/verify-flags/known-flags.txt, as this flag is relevant to one of the kubernetes components. The reason the cluster scripts' author didn't have to make this change is because they're adding this flag for a salt configuration file, see below:

```shell
cat <<EOF >>/etc/salt/minion.d/grains.conf
  kubelet_api_servers: '${KUBELET_APISERVER}'
  cbr-cidr: 10.123.45.0/30
EOF
```

This is very different from what we're doing as they're defining flags not relevant to kubernetes.